app/routes.py

```diff
@@ -1,2 +1,81 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.planet import Planet
+from app import db
 
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message":f"Planet #{planet_id} invalid"}, 400))
+
+    planet = Planet.query.get(planet_id)
+
+    if not planet:
+        abort(make_response({"message":f"Planet #{planet_id} not found"}, 404))
+
+    return planet
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"],
+                        description=request_body["description"],
+                        rings=request_body["rings"])
+    
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(f"Planet {new_planet.name} successfully created", 201)
```

Comment:
👀 Make sure to `jsonify` even plain string responses, otherwise flask will return an HTML response rather than JSON.